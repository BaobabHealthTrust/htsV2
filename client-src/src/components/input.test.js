import React from 'react';
import ReactDOM from 'react-dom';
import Input from './input';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import Enzyme, {
    shallow
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({
    adapter: new Adapter()
});

describe('Component: Input', () => {

    it('should match its empty snapshot', () => {
        const tree = renderer.create( 
            <Input />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    })

})