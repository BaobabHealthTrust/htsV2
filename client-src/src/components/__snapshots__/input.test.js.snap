// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Input should match its empty snapshot 1`] = `
<input
  className={undefined}
  id={undefined}
  onChange={undefined}
  value={undefined}
/>
`;
